xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/HistoryPane.java

```diff
@@ -197,6 +197,7 @@ public ComparePage compare(String fromVersion, String toVersion)
      */
     public int getNumberOfVersions()
     {
+        // TODO: Use the PaginationFilterPane object to get this value.
```

Comment:
Why do you need to add a comment on something that is not changed by the PR?
I was expecting a TODO at https://github.com/xwiki/xwiki-platform/pull/4778/files#diff-73cae6b0d30bfd365b26e25af91a5cdc5c478ad1db1d19d2b5588f09fe77c865R66
(but maybe some other comments are required from the initial commit)
Overall, my point is to make it clear that we need to avoid code duplication and have a single source of truth for selectors related to pagination (though a single PO).